packages/permissions/src/hooks/use-is-authorized/use-is-authorized.spec.tsx

```diff
@@ -103,88 +122,431 @@ const testRender = ({
       <TestComponent
         demandedPermissions={demandedPermissions}
         demandedActionRights={demandedActionRights}
+        demandedDataFences={demandedDataFences}
+        selectDataFenceData={selectDataFenceData}
         shouldMatchSomePermissions={shouldMatchSomePermissions}
       />
     </ApplicationContextProvider>
   );
 
 describe('when only one permission matches', () => {
-  describe('when it should match some permission', () => {
-    it('should indicate being authorized', () => {
-      const { queryByText } = testRender({
-        demandedPermissions: ['ManageCustomers', 'ManageOrders'],
-        allAppliedPermissions: [
-          {
-            name: 'canManageCustomers',
-            value: true,
-          },
-        ],
-        shouldMatchSomePermissions: true,
+  describe('General Permissions', () => {
+    describe('when "shouldMatchSomePermissions=true"', () => {
```

Comment:
Nit: I wouldn't even bind the name of the spec the prop name. It's just "describe(`when all permissions should be matched`"